src/runtime/renderer.ts

```diff
@@ -58,7 +59,7 @@ export function defineRenderHandler(handler: RenderHandler) {
     }
 
     // Send response body
-    return typeof response.body === "string"
+    return typeof response.body === "string" || isStream(response.body)
       ? response.body
       : JSON.stringify(response.body);
```

Comment:
I am wondering do we still need this check still? Not sure why before, but we might directly always return `response.body` and reduce overhead of `isStream` check